catalog/spatial/registry/registry-admin-modules/registry-admin-poller-service/src/main/java/org/codice/ddf/registry/admin/poller/AdminPollerServiceBean.java

```diff
@@ -103,21 +106,28 @@
 
     private static final String PUBLISHED_LOCATIONS = "published-locations";
```

Comment:
Now that this is in the registry app this constant should be changed to use the constant in metacard type.